vector/src/main/java/im/vector/riotx/features/home/room/detail/timeline/TimelineEventController.kt

```diff
@@ -219,8 +220,8 @@ class TimelineEventController @Inject constructor(private val dateFormatter: Tim
                 // Should be build if not cached or if cached but contains mergedHeader or formattedDay
                 // We then are sure we always have items up to date.
                 if (modelCache[position] == null
-                        || modelCache[position]?.mergedHeaderModel != null
-                        || modelCache[position]?.formattedDayModel != null) {
+                    || modelCache[position]?.mergedHeaderModel != null
```

Comment:
this condition is a bit hard to understand

Reply:
it hasn't change in this PR